bin/prettier.cjs

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var pleaseUpgradeNode = require("please-upgrade-node");
+var packageJson = require("../package.json");
+
+pleaseUpgradeNode(packageJson);
+
+function runCli(cli) {
+  return cli.run(process.argv.slice(2));
+}
+
+module.exports.promise = (function () {
+  if (process.env.NODE_ENV !== "production") {
+    var dynamicImport = new Function("module", "return import(module)");
+
+    return dynamicImport("../src/cli/index.js").then(runCli);
+  }
```

Comment:
Just a question, why are we use dynamic import when only in non-production env?

Reply:
"../src/cli/index.js" is an ESM module, it can't be `require()`d.

Reply:
I'm not comfortable with current solution either. Got a better idea?

Reply:
This confusing you maybe because you don't know this https://github.com/prettier/prettier/blob/7544c32c538e61be3d9bddb6ea4e64935d5b0200/scripts/build/bundler.mjs#L89

Reply:
I got it, but I have no better idea..